charts/spv-wallet-stack/charts/spv-wallet/templates/secret.yml

```
@@ -15,3 +15,7 @@ data:
     {{- $token = (get $secretData "callback_token") | default (randAlphaNum 32 | b64enc) }}
   {{- end }}
   callback_token: {{ $token | quote }}
+
+  {{- if .Values.config.paymail.beef.block_headers_service_auth_token }}
+  block-headers-service-auth-token: {{ .Values.config.paymail.beef.block_headers_service_auth_token | b64enc }}
```

For the sake of being safe regardless the input lets add quotation
```suggestion
  block-headers-service-auth-token: {{ .Values.config.paymail.beef.block_headers_service_auth_token | b64enc | quote }}
```